service/src/test/java/uk/gov/hmcts/reform/fpl/service/orders/history/SealedOrderHistoryServiceTest.java

```diff
@@ -75,6 +76,10 @@
     private static final DocumentReference SEALED_PDF_DOCUMENT = mock(DocumentReference.class);
```

Comment:
careful with these static mocks, they remember stubbings from previous runs

Reply:
it's ok to have mock constants. The problem is in case we do a when().then() on a constant, that is wrong.